rotkehlchen/db/eth2.py

```diff
@@ -191,11 +220,16 @@ def get_active_validator_indices(self, cursor: 'DBCursor') -> set[int]:
         )
         return {x[0] for x in cursor}
 
-    def get_exited_validator_indices(self, cursor: 'DBCursor') -> set[int]:
+    @staticmethod
+    def get_exited_validator_indices(cursor: 'DBCursor', validator_indices: Collection[int] | None) -> set[int]:  # noqa: E501
```

Comment:
edit the docstring with the new optional argument you added. Just explain what it does in an one liner too